### ISSUE TITLE
未定义的相机内参 intrinsics_curr 与 pred_bidir_depth 分支的可用性/健壮性

### ISSUE THREAD

**Issue by H-Tenets** [LianpoWang/Stereo-DICNet2#27]
在用于 depth/refine 的分支中，有代码片段使用 intrinsics_curr（和 pose）： if pred_bidir_depth and refine_iter_idx == 0: intrinsics_curr = intrinsics_curr.repeat(2, 1, 1) pose = torch.cat((pose, torch.inverse(pose)), dim=0) ... flow_from_depth = compute_flow_with_depth_pose(1. / flow.squeeze(1), intrinsics_curr, extrinsics_rel=pose) 这里并没有在 forward 的参数或函数范围内定义 intrinsics_curr（也没有检查 pose 是否为 None），因此在执行 pred_bidir_depth 或 depth 路径时会抛出 NameError 或使用未初始化的变量。请确认：
intrinsics_curr 应该从何处传入（forward 参数？类属性？），并在文档/代码中明确；
在进入 pred_bidir_depth、depth 或 refine 相关分支前应校验 pose/intrinsics 的有效性并给出清晰的错误提示或默认行为；
如果这些分支在当前实现中未测试或可选，应添加断言或把相关参数设为 forward 的显式输入以避免隐式依赖。
